database/migrations/2020_09_03_135257_create_invitations_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateInvitationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('invitations', function (Blueprint $table) {
+            $table->id();
+            $table->string('email')->unique();
+            $table->uuid('invitation_token')->unique()->nullable();
```

Comment:
data format is wrong

Reply:
https://github.com/THM-Health/PILOS/pull/23/commits/1a33a7aa89dddde61afd7c0994421ca8d1e20bfb changed to ``(string,36)``